docs/build/credentials.md

```diff
@@ -30,8 +30,95 @@ authenticate with your app (e.g., `username`, `api_key`).
 If your app is not listed in the Adaptors section, then inspect your app's API
 documentation to see what is required for "authentication". You can then create
 a `Raw JSON` Credential in OpenFn to define whatever credential inputs are
-require (e.g., `{"api_key": "ADD-your-let", "url": "add-url"}`).
+required. E.g.:
 
-Some systems (Salesforce, OpenMRS, DHIS2) require an instanceUrl, host, or
-ApiUrl. Leave off the final "/" in these Urls: `https://login.salesforce.com` or
-`http://demo.openmrs.org/openmrs` or `https://play.dhis2.org`.
+```json
+{
+  "api_key": "someSecretKey",
+  "base_ur": "https://example.com/api/v2"
+}
+```
+
+Note that some systems (Salesforce, OpenMRS, DHIS2) require an instanceUrl,
+host, or ApiUrl. While most adaptors will handle a "trailing slash" in a URL
+gracefully, when in doubt you should leave it off. For example:
+
+- prefer `https://login.salesforce.com` to `https://login.salesforce.com/`,
+- use `http://demo.openmrs.org/openmrs` instead of
+  `http://demo.openmrs.org/openmrs/`,
+- and write `https://play.dhis2.org` rather than `https://play.dhis2.org/`.
+
+### Use Oauth2 credentials
+
+If Oauth2 _clients_ have been configured on your OpenFn instance, you can use
+them create Oauth credentials. These all work the same way:
+
+1. First you pick an Oauth credential type from the "New Credential" interface.
+2. Then you give it a name.
+3. You optionally select additional "scopes" that you want to use. (See the
+   in-app link to third-party documentation on scopes, depending on what
+   application you're using.)
+4. And then click "Sign in with \_\_\_\_\_\_".
+
+You will be asked by the third-party application to verify your identity and
+that you'd like to use Oauth. Once you accept, back on OpenFn you'll be able to
+save and use your new credential just like any other.
+
+:::tip
+
+If you don't see an Oauth credential type available for your app, contact the
```

Comment:
Maybe to reword this a bit, because it's for deployed instance so it should start with that 
>If you're using a deployed instance of OpenFn and can't find the OAuth credential type for your app, please contact the superuser responsible for your instance setup and request them to set up OAuth clients for your application

Reply:
addressed in next commit